config/locales/devise/en.yml

```diff
@@ -179,22 +185,27 @@ en:
       two_factor_choice_intro: login.gov makes sure you can access your account by
         adding a second layer of security.
       two_factor_choice_cancel: "‹ Choose another option"
-      two_factor_login_choice: Secure your account
-      two_factor_login_choice_options:
-        voice: Phone call
-        voice_info: Get your security code via phone call.
-        sms: Text message / SMS
-        sms_info: Get your security code via text message / SMS.
-        auth_app: Authentication application
-        auth_app_info: Set up an authentication application to get your security code
-          without providing a phone number.
-        piv_cac: Government employees
-        piv_cac_info: Use your PIV/CAC card to secure your account.
+      two_factor_login: Secure your account
+      two_factor_login_add_methods_intro: You set these up when you created your account
+      two_factor_login_add_methods_title: Choose additional methods to secure your
+        account
+      two_factor_login_options:
+        voice: Automated phone call
+        voice_info: Get security code via phone call (North America phone numbers
+          only).
+        sms: Text message
+        sms_info: Get security code via text message.
+        auth_app: Authentication app
+        auth_app_info: Use your authentication application to get security code.
+        piv_cac: Government employee ID
+        piv_cac_info: Use your PIV/CAC card instead of a security code.
         personal_key: Personal Key
-        personal_key_info: Use the 12 character personal key you used at account creation.
-      two_factor_login_choice_intro: login.gov makes sure you can access your account
-        by adding a second layer of security.
-      two_factor_login_choice_cancel: "‹ Choose another option"
+        personal_key_info: Use the 16 character personal key you received at account
+          creation.
+      two_factor_login_options_title: Select your security option
+      two_factor_login_options_link_text: Choose another security option
+      two_factor_login_intro: You set these up when you created your account
+      two_factor_login_cancel: "‹ Choose another option"
```

Comment:
I know a lot of these were here before, but maybe we can use this as an opportunity to move them somewhere else because these are not related to Devise. There is a bug currently in our `i18n-tasks` config where all Devise translations are being ignored, and so the specs will never tell us if we have missing or unused specs in this file. I noticed this a while back and haven't gotten around to fixing it. I opened an issue just now so we can track it.

Reply:
Are you sure that is still an issue?  I'm pretty sure I culled some unused translations from devise using that script. I agree it should be moved from devise.  However, I wasn't happy with the files that partitioned them by function (ie links/en.yml, instructions/en.yml, etc.) I'd prefer the files were scoped at the feature level.

Reply:
I'm sure it's an issue. For example, `phone_voice_info_html` in this file is not used in the app, yet the specs are not flagging it. Try running `i18n-tasks unused`